### ISSUE TITLE
Errors installing on Ubuntu 16.04

### ISSUE THREAD

**Issue by ArvidR** [scylladb/scylla#3518]
*Installation details*
Scylla version: Download: 2.1
OS: Ubuntu 16.04 LTS

Could not resolve host repositories.scylladb.com
Unable to locate package Scylla

See attached screenshot.

![scyllainstallerror](https://user-images.githubusercontent.com/14226742/41429697-bceadda2-7005-11e8-8385-3d03b16635ca.png)

**Comment by tzach** [comment]
@amnonh can you please take a look?

**Comment by tzach** [comment]
@ArvidR I suspect networking issue 
can you please check
```
ping repositories.scylladb.com
```

**Comment by ArvidR** [comment]
Looks OK:

**Comment by ArvidR** [comment]
Something was corrected just recently.  I am now able to continue 
installing.
Thanks!

**Comment by ArvidR** [comment]
Tzach,
I made no changes to the scylla.yaml file.  2 more errors:
I'm looking at the /etc/scylla/cassandra/cassandra-env.sh.  Port 7199 is 
specified as the JMX_PORT.  I also made no changes here.
Any ideas?

Thanks!




Jeigu nėra pareikšta kitaip: / Unless stated otherwise above:
UAB IBM Lietuva 
Šeimyniškių g. 3 
LT-09312 Vilnius
Lietuva 
Reg. Nr.: 111620231

**Comment by ArvidR** [comment]
Any luck?
Arvid Raulinaitis
IT Architect
Global Business Services
+370 8 6 9483828
arvid.raulinaitis@ibm.com
Leave Plan: July 2-6 Limited Availability



From:   Arvid Raulinaitis/Lithuania/IBM
To:     scylladb/scylla 
<reply@reply.github.com>
Cc:     Mention <mention@noreply.github.com>, scylladb/scylla 
<scylla@noreply.github.com>
Date:   06/18/2018 11:03 AM
Subject:        Re: [scylladb/scylla] Errors installing on Ubuntu 16.04 
(#3518)


Tzach,
I made no changes to the scylla.yaml file.  2 more errors:
I'm looking at the /etc/scylla/cassandra/cassandra-env.sh.  Port 7199 is 
specified as the JMX_PORT.  I also made no changes here.
Any ideas?

Thanks!




Jeigu nėra pareikšta kitaip: / Unless stated otherwise above:
UAB IBM Lietuva 
Šeimyniškių g. 3 
LT-09312 Vilnius
Lietuva 
Reg. Nr.: 111620231




Jeigu nėra pareikšta kitaip: / Unless stated otherwise above:
UAB IBM Lietuva 
Šeimyniškių g. 3 
LT-09312 Vilnius
Lietuva 
Reg. Nr.: 111620231
